app/server.py

```diff
@@ -49,7 +49,7 @@ def search(search_engine):
             err = [400, 'Not Found - missing query', qformat]
             return bad_request(err)
 
-        if engine[0] == 'q':
+        if engine[0] in ['q', 't']:
```

Comment:
Putting this logic in two different files can be the source of future bugs.  Please encapsulate this logic in one file or the other but not in both.